File: 1720-crawler-log-folder/crawler-log-folder.c
int minOperations(char** logs, int logsSize) {
    int i,c=0;
    for(i=0;i<logsSize;i++)
    {
        if(strcmp(logs[i],"./")==0)
        continue;
        if(strcmp(logs[i],"../")==0)
        c=(c>0) ? c-1 : 0;
        else
        ++c;
    }
    return c;
}

File: 1720-crawler-log-folder/crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        int c=0;
        for(final String ch:logs)
        {
            if(ch.equals("./"))
            continue;
            if(ch.equals("../"))
            c=Math.max(0,c-1);
            else
            ++c;
        }
        return c;
    }
}